lib/msal-browser/src/app/IPublicClientApplication.ts

```diff
@@ -56,7 +56,7 @@ export interface IPublicClientApplication {
     logoutRedirect(logoutRequest?: EndSessionRequest): Promise<void>;
     logoutPopup(logoutRequest?: EndSessionPopupRequest): Promise<void>;
     ssoSilent(request: SsoSilentRequest): Promise<AuthenticationResult>;
-    getTokenCache(): ITokenCache;
+    getTokenCache(): TokenCache;
```

Comment:
I would like to see this also removed as part of this

Reply:
Done!